hm6_1.py

```diff
@@ -0,0 +1,10 @@
+""" Task 1: Перевести строку в список
+
+"Robin Singh" => ["Robin”, “Singh"]
+
+"""
+
+TXT = "Robin Singh"
+
+print(TXT, type(TXT))
+print(TXT.split(), type(TXT.split()))
```

Comment:
the same issue as in HW5

Reply:
Understand your message, will fix, thank you.